docs/specs/editorconfig-in-the-compiler.md

```diff
@@ -0,0 +1,110 @@
+All-capital words are interpreted per [RFC 2119](https://www.ietf.org/rfc/rfc2119.txt).
+
+# Compiler Command Arguments
+
+The compiler *MUST* receive a new switch, `/editorconfig`, which accepts a single
+file which is an .editorconfig file that may apply to one or more files in the
+compilation. Multiple files are passed with multiple `/editorconfig` switches,
+the same way you can pass multiple references with /reference or multiple analyzers
+with /analyzer. There is no short form of the argument. Files passed with relative
+file names are resolved relative to the current working directory the same
+as source files.
+
+Examples:
+
+```
+    csc.exe /editorconfig:.editorconfig
+    csc.exe /editorconfig:.editorconfig /editorconfig:..\.editorconfig
+    csc.exe /editorconfig:.editorconfig /editorconfig:Subdirectory\.editorconfig
+```
+
+It is expected that the compiler is given the .editorconfig files with casing that
+matches the underlying casing of the file system. The compiler *MUST NOT* do any case correction. So
+
+```
+    csc.exe /editorconfig:Subdirectory\.editorconfig /editorconfig:SubDirectory\.editorconfig
+```
+
+is a valid input to the compiler. The compiler *MAY* choose to issue a warning if two paths given
+are exactly equal by ordinal in a case-sensitive way.
+
+If multiple .editorconfigs apply to a given file (the precise definition of
```

Comment:
It seems like, for the majority of cases, users would want `.editorconfig` to "just work" (as it does in the IDE and everywhere else that supports it -- practically all modern editors have a plugin).

As such, it would seem there should be a special mode that indicates "ignore normal rules and do things explicitly" and the default behavior (from just passing /editorconfig) would follow the normal resolution rules:
> When opening a file, EditorConfig plugins look for a file named .editorconfig in the directory of the opened file and in every parent directory. A search for .editorconfig files will stop if the root filepath is reached or an EditorConfig file with root=true is found.
> 
> EditorConfig files are read top to bottom and the closest EditorConfig files are read last. Properties from matching EditorConfig sections are applied in the order they were read, so properties in closer files take precedence.

This is what I (and what I imagine anyone else familiar with .editorconfig) would expect to happen).